### ISSUE TITLE
Option to remove the Mobile data and Wifi from the widget.

### ISSUE THREAD

**Issue by at-pyrix** [akaMrNagar/Mindful#103]
**Have you checked it on [Project Progress](https://github.com/users/akaMrNagar/projects/1) ?**

yes

**Is your feature request related to a problem? Please describe.**
I am using lawnchair and whenever I make the widget small enough, the wifi and mobile data usage go away but the title remains. Regardless, I don't even want to see data usage on my screen.

![Screenshot_20241023-212220_Lawnchair](https://github.com/user-attachments/assets/bde2dcb1-b894-4fd3-a1c4-01a2088720f0)

**Describe the solution you'd like**
An option to remove the data usage from the widget would be fantastic! 

Thanks!

**Comment by at-pyrix** [comment]
also the 12 hr screen time is due to settings for some reason 😅

**Comment by at-pyrix** [comment]
![Screenshot_20241025-082221_Mindful](https://github.com/user-attachments/assets/8bf88006-4f77-4056-b1ca-785c22ecc8b6)

It's a bug I guess.

**Comment by akaMrNagar** [comment]
Yeah i have to check it out. But i am out of town till 5 November. Till then keep an eye
